### PR TITLE
Add more documentation and corrections for gma.lua, gma.canbus, gma.network, gma.show and gma.show.getobj

### PR DESCRIPTION
**Pull request by gutasiadam**
Added documentation and corrections for gma.lua, gma.canbus, gma.network, gma.show and gma.show.getobj.

### CODE REVIEW

**Comment by Hobadee** [comment]
Thanks for this!

I did notice some grammar and spelling errors, so those will need to be cleaned up.  (It sounds like English might not be your first language so no worries, I can do this a bit later when I'm not on my phone.)

Also, I don't have access to a physical MA anymore to test with.  Everything looks like it makes sense, but can you please confirm how you got this info?  (Experimentation is fine, I just want it on the public record.)

**Comment by gutasiadam** [comment]
Thank you for your feedback! 
I could soon fix the spelling errors as well. I am unsure whether a new merge request is required for another commit.

The information I added to the documentation is based **solely on experiments**. Currently, I only have access to a command wing with onPc. Based on the concept of how the onPc version emulates real MA, I believe these commands should work just fine.

File: gma.lua [hunk 1]
```diff
@@ -53,28 +53,28 @@ end
...
+--@tparam table export_data Table object to export
 function gma.export_csv(filename,export_data)
 end
 
----Export a table as a JSON file?
+---Export a table as a JSON file
 --
---@tparam string filename Filename to export to???
---@tparam table export_data Table object to export???
+--@tparam string filename Filename to export to. Exports to \MA Lighting Technologies\grandma\gma2_V_<YourVersion>\temp
```
Comment (Hobadee) [inline gma.lua]: Is this tested on a console, onPC, or both?

File: gma.lua [hunk 2]
```diff
@@ -53,28 +53,28 @@ end
...
+--@tparam table export_data Table object to export.
 function gma.export(filename,export_data)
 end
 
----Export a table as a CSV file?
+---Export a table as a CSV file
 --
---@tparam string filename Filename to export to???
---@tparam table export_data Table object to export???
+--@tparam string filename Filename to export to. Exports to \MA Lighting Technologies\grandma\gma2_V_<YourVersion>\temp
```
Comment (Hobadee) [inline gma.lua]: Is this tested on a console, onPC, or both?

File: gma.lua [hunk 3]
```diff
@@ -53,28 +53,28 @@ end
 function gma.git_version()
 end
 
----Export a table as a file???
+---Export a table as a file. By default, MA uses XML as a file type.  Check @{gma.export_csv()} and @{gma.export_json()} for other options.
 --
---@tparam string filename Filename to export to???
---@tparam table export_data Table object to export???
+--@tparam string filename Filename to export to. Exports to \MA Lighting Technologies\grandma\gma2_V_<YourVersion>\temp
```
Comment (Hobadee) [inline gma.lua]: Is this tested on a console, onPC, or both?

Comment (gutasiadam) [reply gma.lua]: Forgot to add, that it is tested only on onPC version.

**Comment by Hobadee** [comment]
I reviewed grammar, spelling, and formatting.  Good to merge.

**Pull request merged**